packages/transport-http/src/http-request.js

```diff
@@ -169,12 +187,18 @@ export async function httpRequest({
 
           res.on("end", () => {
             try {
-              responseBody = JSON.parse(responseBody.join(""))
+              responseBody =
+                responseBody && responseBody.length
+                  ? JSON.parse(responseBody.join(""))
+                  : null
               if (
                 res?.statusCode &&
                 (Number(res?.statusCode) < 200 ||
                   Number(res?.statusCode) >= 300)
               ) {
+                if (res.statusCode == 404) {
```

Comment:
Same here, I'm afraid. Triggers with correct endpoint

Reply:
This one seems to work for me when I test it with rest-testnet on node